design/ceph/object/store.md

```diff
@@ -103,6 +108,8 @@ The RGW service can be configured to listen on both http and https by specifying
     instances: 1
 ```
 
+- `dnsNames`: A list of hostnames to use for accessing the bucket directly like a subdomain in the endpoint. For example if the default service endpoint `rook-ceph-rgw-my-store.rook-ceph.svc:80` added and the object store contains a bucket named `sample`, then the s3 [virtual host style](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html) would be `http://sample.rook-ceph-rgw-my-store.rook-ceph.svc:80`. More details about the feature can be found in [Ceph Documentation](https://docs.ceph.com/en/latest/radosgw/s3/commons/#bucket-and-host-name)
```

Comment:
The spec here is still referencing the field as though users should input `http://<hostname>:port`. Please fix

Reply:
Also, let's move this to be with the virtual host style access for buckets below. It seems strange to me to have this in two places, especially when this section gives the impression that `dnsNames` is a top-level field, rather than nested under `hosting`.